src/coreclr/interpreter/intrinsics.cpp

```diff
@@ -63,6 +70,15 @@ NamedIntrinsic GetNamedIntrinsic(COMP_HANDLE compHnd, CORINFO_METHOD_HANDLE comp
         // Every intrinsic except IsSupported is PNSE in interpreted-only mode.
         return NI_Throw_PlatformNotSupportedException;
     }
+    else if (HAS_PREFIX(namespaceName, "JIT.HardwareIntrinsics"))
```

Comment:
JIT.HardwareIntrinsics are tests. Why are recognizing test methods as intrinsics? It does not sound right.

>